src/lib/components/explorer/FacetItem.svelte

```diff
@@ -97,7 +97,6 @@
     name={facet.name}
     value={facet}
     {checked}
-    disabled={facet.count === 0}
```

Comment:
I think this should be
`disabled={facet.count === 0 && !checked}`

Reply:
I'll check with Emily. It's a good question: Should it be clickable if the value is 0 (not having previously been clicked).

Reply:
The story ACs list the behavior you noted. Let me update. :)